include/miopen/miopen.h

```diff
@@ -5793,6 +5795,36 @@ MIOPEN_EXPORT miopenStatus_t miopenSumForward(miopenHandle_t handle,
 // CLOSEOUT SUM DOXYGEN GROUP
 #endif
 
+#ifdef MIOPEN_BETA_API
+// Padreflection APIs
+/** @addtogroup padreflection
+ *
+ *  @{
+ */
+/*! @brief Add padding by the reflection of the tensor
+ *
+ * @param handle                   MIOpen handle (input)
+ * @param nanPropagation           Nan number propagation mode (input)
+ * @param workspace                Address of the allocated workspace data (input)
+ * @param workspaceSizeInBytes     Size in bytes of the allocated workspace data (input)
+ * @param xDesc                    Tensor descriptor for data input tensor x (input)
+ * @param x                        Data tensor x (input)
+ * @param dim                      Dimensions to sum. (input)
+ * @param yDesc                    Tensor descriptor for output data tensor y (input)
+ * @param y                        Data tensor y (output)
+ * @return                         miopenStatus_t
+ */
+MIOPEN_EXPORT miopenStatus_t miopenPadReflection(miopenHandle_t handle,
+                                              const miopenTensorDescriptor_t xDesc,
+                                              const void* x,
+                                              const miopenTensorDescriptor_t yDesc,
+                                              void* y,
+                                              const std::vector<size_t> padding);
```

Comment:
Do not use vector in miopen.h. This is a rule in MIOpen. Receive pointer and number input from the user.

Reply:
I have change the implementation from vector to pointer size_t * in https://github.com/moreh-dev/MIOpen/pull/7/commits/a0ef46c20a734c34e1342c87b9cfb9f6d40e5098